### ISSUE TITLE
Unknown error

### ISSUE THREAD

**Issue by aurany** [dbt-labs/dbt-adapter-tests#35]
Hi,

I'm trying to bump the ibmdb2 adapter deps from dbt-core 0.19.1 to 1.0.3. The test is spitting out some hard-to-understand errors. Does anyone know what this unknown error is? Ideas on how to debug?

```
Rasmuss-MBP:dbt-ibmdb2 rasmus$ poetry run pytest -v --log-level DEBUG test/dbt-adapter-tests.dbtspec
============================================================================== test session starts ==============================================================================
platform darwin -- Python 3.9.12, pytest-6.2.5, py-1.11.0, pluggy-1.0.0 -- /Users/rasmus/projects/dbt_dev/dbt-ibmdb2/.venv/bin/python
cachedir: .pytest_cache
rootdir: /Users/rasmus/projects/dbt_dev/dbt-ibmdb2
plugins: dbt-adapter-0.5.1
collected 9 items                                                                                                                                                               

test/dbt-adapter-tests.dbtspec::test_dbt_empty PASSED                                                                                                                     [ 11%]
test/dbt-adapter-tests.dbtspec::test_dbt_base FAILED                                                                                                                      [ 22%]
test/dbt-adapter-tests.dbtspec::test_dbt_ephemeral FAILED                                                                                                                 [ 33%]
test/dbt-adapter-tests.dbtspec::test_dbt_incremental FAILED                                                                                                               [ 44%]
test/dbt-adapter-tests.dbtspec::test_dbt_snapshot_strategy_timestamp FAILED                                                                                               [ 55%]
test/dbt-adapter-tests.dbtspec::test_dbt_snapshot_strategy_check_cols FAILED                                                                                              [ 66%]
test/dbt-adapter-tests.dbtspec::test_dbt_data_test FAILED                                                                                                                 [ 77%]
test/dbt-adapter-tests.dbtspec::test_dbt_schema_test FAILED                                                                                                               [ 88%]
test/dbt-adapter-tests.dbtspec::test_dbt_ephemeral_data_tests FAILED                                                                                                      [100%]

=================================================================================== FAILURES ====================================================================================
____________________________________________________________________________ usecase: test_dbt_base _____________________________________________________________________________
Unknown error: assert 1 == 0
  +1
  -0 in test index 1 (item_type=run_results)
----------------------------------------------------------------------------- Captured stdout call ------------------------------------------------------------------------------
Executing step 1/15
```

**Comment by jtcohen6** [comment]
Hey @aurany, I'm not sure of the exact issue you might be running into... Have you upgraded to use the latest available version (`pytest-dbt-adapter==0.6.0`), for compatibility with dbt-core v1.0? https://github.com/dbt-labs/dbt-adapter-tests/pull/32

One other thing to call out: We're planning to replace this package (https://github.com/dbt-labs/dbt-adapter-tests/issues/36) with a newer framework that looks and feels a bit more like `pytest`. If you're up for giving that a go, we have an initial version available, and [some WIP docs](https://deploy-preview-1263--docs-getdbt-com.netlify.app/docs/contributing/testing-a-new-adapter).

**Comment by aurany** [comment]
Thanks @jtcohen6! This error is now gone using 0.6.0. Now I have to deal with the quoting error again (https://github.com/dbt-labs/dbt-adapter-tests/issues/16) :)
